steps/push-changes.ps1

```diff
@@ -14,6 +14,15 @@ Push-Location $RepoPath
 
 try {
 
+    Write-Output "Pulling"
+    $Command = "git pull origin $Branch"
```

Comment:
Hi @ben51degrees !
The small nitpick that Alex has is to consider `git pull --rebase` to avoid messy git history.

Reply:
Thanks, I'll check this and update before completing the PR.